components/results_table/wikis/starcraft2/results_table_custom.lua

```diff
@@ -164,6 +160,11 @@ function CustomResultsTable._storeAllKill(args)
 	})
 end
 
+---Builds a map row for the all kill match
+---@param args table
+---@param prefix string
+---@param match any
```

Comment:
don't have a class for div tables (they aren't on the git) so to not get a `undefined field 'row'` in line 195 using any here